src/sentry/incidents/logic.py

```diff
@@ -525,14 +527,17 @@ def create_alert_rule(
         actor = owner
 
     with transaction.atomic(router.db_for_write(SnubaQuery)):
+        # NOTE: `create_snuba_query` constructs the postgres representation of the snuba query
+        # TODO: ensure created SnubaQuery has all relevant fields for constructing query in snuba on the fly
+        # (constructed in `subscribe_projects_to_alert_rule`)
```

Comment:
It does. All the `QuerySubscription` contains is the `project_id`, and in the future the `query_extra`